src/pages/Dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -19,6 +20,7 @@ const paddingTop = {
   md: '4.5rem',
 }
 const marginTop = { base: 0, md: '-4.5rem' }
+const borderBottomWidth = { base: 0, md: 1 }
```

Comment:
No border below the header on mobile i.e 

<img width="944" alt="image" src="https://github.com/user-attachments/assets/107599e7-d6a9-4fb9-a32f-c18fc63c87a9" />
<img width="428" alt="image" src="https://github.com/user-attachments/assets/d5ccd7bd-b5fb-4ec0-90f7-9204add16822" />